indico/htdocs/js/indico/modules/registration/registration.js

```diff
@@ -47,4 +48,158 @@
             $('#payment-summary').effect('highlight', 800);
         });
     }
+
+    var editor = $('.registrations');
+
+    function flagChanged() {
+        editor.trigger('ajaxDialog:setData', [true]);
+    }
+
+    function setupStaticURLGeneration() {
+        $('.js-static-url').on('click', function(event) {
+            var $this = $(this);
+            $.ajax({
+                method: 'POST',
+                url: $this.data('href'),
+                success: function(result) {
```

Comment:
`error: handleAjaxError` and possibly `complete: IndicoUI.Dialogs.Util.progress()` if you want a "loading.." dialog